docs/compatibility.md

```diff
@@ -785,7 +785,7 @@ The GPU implementation of `approximate_percentile` uses
 [t-Digests](https://arxiv.org/abs/1902.04023) which have high accuracy, particularly near the tails of a
 distribution. Because the results are not bit-for-bit identical with the Apache Spark implementation of
 `approximate_percentile`, this feature is disabled by default and can be enabled by setting
-`spark.rapids.sql.expression.ApproximatePercentile=true`.
+`spark.rapids.sql.incompatibleOps.enabled=true`.
```

Comment:
This says that approx percentile is disabled by default and can be enabled by setting `incompatibleOps.enabled=true`, yet the configs.md document states that it _is enabled by default_ and does not mention anything about being incompatible with Spark's implementation.  Looks like the meta class for approx percentile in GpuOverrides should have been marked with `.incompat` and text explaining what is not 100% compatible with Spark.